imagery/i.signatures/main.c

```diff
@@ -0,0 +1,225 @@
+/****************************************************************************
+ *
+ * MODULE:       i.signatures
+ * AUTHOR(S):    Maris Nartiss - maris.gis gmail.com
+ * PURPOSE:      Manages signature files of imagery classifiers
+ *
+ * COPYRIGHT:    (C) 2023 by Maris Nartiss and the GRASS Development Team
+ *
+ *               This program is free software under the GNU General Public
+ *               License (>=v2). Read the file COPYING that comes with GRASS
+ *               for details.
+ *
+ *****************************************************************************/
+#include <stdlib.h>
+#include <string.h>
+#include <stdbool.h>
```

Comment:
Is including stdbool.h necessary in this file?

Reply:
Not any more. Removed.